python/ml4ir/applications/ranking/tests/test_base.py

```diff
@@ -43,7 +55,7 @@ def setUp(
         random.seed(123)
```

Comment:
seed should probably be configurable via args, right?

Reply:
This is in only for the tests. It is configurable through command line arguments when you are running the actual pipeline itself.